pkg/parser/extended_syntax_parser.y

```diff
@@ -76,11 +76,12 @@ type StandardSelect struct {
 }
 
 type TrainClause struct {
-	Estimator  string
-	TrainAttrs Attributes
-	Columns    columnClause
-	Label      string
-	Save       string
+	Estimator       string
+	TrainAttrs      Attributes
+	Columns         columnClause
+	Label           string
+	PreTrainedModel string
```

Comment:
parser and IR represents the parsed SQL statement information, `PreTrainedModel` is more likely to be how this `USING` field will be used, not representing what the parse the field is. Maybe change it to `TrainUsing` ?

Reply:
Sounds good. Done.